source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,126 @@
+This method takes a single parameter, the host ID from which to copy the configuration (`Host0`). 
+
+### Limitations
+
+* You can copy host configurations only within the same cluster. 
+
+* You cannot copy network configurations that contain static IP addresses: 
+This sets the boot protocol in the resulting network to `none`.
+
+* Copying a configuration to a host with the same interface names but different physical network connections will
+produce a bad configuration. Only copy configurations to hosts that have the same interface names and physical 
+network connections. 
+
+* The destination host has to have an equal or greater number of interfaces as the original host. Otherwise,
+the feature cannot determine what to do.
+
+* The `ovirtmgmt` network stays on the original interface which is excluded from the operation. 
+
+
+### Implementation
+
+The engine has a new method, CopyNetworks, that copies the network configuration of the source host (SourceHost)
+and uses it to overwrite the network configuration of the destination host (DestinationHost).
+
+This method takes two parameters, the IDs of the SourceHost and DestinationHost.
+
+Given the parameters, the method validates that the following items are correct:
+
+* Both hosts are in the same cluster.
+* The destination host has an equal or higher number of interfaces than the source host.
+
+(NOTE: These interface names and physical connections to the datacenter must be identical. Otherwise, this sorting
+process produces a bad network configuration on the destination host.)
+
+The method prepares a set of network attachment definitions for the DestinationHost by doing the following:
+
+* Lists the interfaces on the SourceHost that don't have bonds.
+* Lists all of the interfaces on the DestinationHost.
+* Sorts both lists by name.
+* Excludes interface/bonds connected to the ovirtmgmt network.
+* Pairs all of the bond slaves from the SourceHost with their corresponding partners on the DestinationHost.
+* Creates bond definitions in a configuration for the DestinationHost.
+* Retrieves a list of all interfaces with bonds from SourceHost.
+* Excludes bond slaves from both lists.
+* Shortens the DestinationHost list to match the same length as SourceHost list.
+* Copies the network attachments for all interfaces in SourceHost list.
+* Walks through the new network attachments list and updates any static IP address configurations.
+* Sends HostSetupNetworks with new attachments and bonds for DestinationHost to complete everything in a single
+ transaction.
+* If there is an error, it reverts the configuration changes.
+
+The Admin Portal has a new button called Copy Host Networking. Clicking the button opens a dialog window. There, 
+you can select the source host from which to copy the configuration.
+
+
+### Future work
+
+To improve usability the biggest limitation is network mapping of interfaces with different name. 
```

Comment:
Not very clear, maybe just: 
The biggest limitation is network mapping of interfaces with different name.

Reply:
Done